velox/exec/FilterProject.cpp

```diff
@@ -184,22 +164,28 @@ RowVectorPtr FilterProject::getOutput() {
     if (!allRowsSelected) {
       rows->setFromBits(filterEvalCtx_.selectedBits->as<uint64_t>(), size);
     }
-    project(*rows, evalCtx);
+    project(*rows, evalCtx, results);
   }
 
   return fillOutput(
-      numOut, allRowsSelected ? nullptr : filterEvalCtx_.selectedIndices);
+      numOut,
+      allRowsSelected ? nullptr : filterEvalCtx_.selectedIndices,
+      results);
 }
 
-void FilterProject::project(const SelectivityVector& rows, EvalCtx& evalCtx) {
+void FilterProject::project(
```

Comment:
Since 'results' are not reused, it would be simpler to return results. This would make code more readable.